src/index.js

```diff
@@ -54,11 +55,10 @@ export default class PreloadPlugin extends Plugin {
 		// initial preload of links with [data-swup-preload] attr
 		swup.preloadPages();
 
-		// do the same on every content replace
-		swup.on('contentReplaced', this.onContentReplaced);
```

Comment:
Could you put these two lines back in? :)

Reply:
In my first commit i removed them, in my second commit i put them back inside the if (this.options.preloadInitialPage)

Reply:
Actually, I wasn't wrong! These two lines have to go back in:

```js
// do the same on every content replace
swup.on('contentReplaced', this.onContentReplaced);
```
This will make sure that this code runs every time swup replaces the content:
```js
onContentReplaced = () => {
	this.swup.preloadPages();
};
```

Reply:
Oh sorry! I totally missed this